lib/redis_client/cluster/command.rb

```diff
@@ -101,6 +118,41 @@ def determine_first_key_position(command) # rubocop:disable Metrics/CyclomaticCo
         end
       end
 
+      # IMPORTANT: this determines the last key position INCLUSIVE of the last key -
+      # i.e. command[determine_last_key_position(command)] is a key.
+      # This is in line with what Redis returns from COMMANDS.
+      def determine_last_key_position(command, keys_start) # rubocop:disable Metrics/AbcSize
```

Comment:
I didn't end up needing this method but it was tricky enough to write, and a natural enough analogue of `determine_first_key_position`, that i'm almost tempted to suggest it be committed in case it's needed in the future.